docs/generated/packages/nx/documents/init.md

```diff
@@ -5,7 +5,7 @@ description: 'Adds Nx to any type of workspace. It installs nx, creates an nx.js
 
 # init
 
-Adds Nx to any type of workspace. It installs nx, creates an nx.json configuration file and optionally sets up remote caching. For more info, check https://nx.dev/recipes/adopting-nx.
+Adds Nx to any type of workspace. It installs nx, creates an nx.json configuration file and optionally sets up remote caching. For more info, check out the [adopting Nx recipe](https://nx.dev/recipes/adopting-nx).
```

Comment:
This is automatically generated from packages/nx/src/command-line/init/command-object.ts. The nx documentation command will sync to the MD file. However, since this affects the CLI output, we cannot show Markdown.